sql-scripts/create-authentication.sql

```diff
@@ -0,0 +1,37 @@
+USE TheITCrowd_MiaG;
+
+CREATE TABLE Users (
+    userId INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    email VARCHAR(50) UNIQUE NOT NULL,
+    passwordHash TEXT NOT NULL,
+    firstName VARCHAR(50) NOT NULL,
+    lastName VARCHAR(50) NOT NULL,
+    accessRight ENUM('Dummy', 'Blocked', 'View', 'Admin') DEFAULT 'View' NOT NULL
+);
+
+DROP PROCEDURE IF EXISTS insertFakeUserData;
```

Comment:
Is there a reason for using a transaction? None of the other SQL scripts are using them

Reply:
Just thinks its a safer implementation. If one test data fails I thought it would be best to fail all the inserts.